### PR TITLE
game of thrones quotes

### LINKED ISSUE #76: [idea]: Game of Thrones Quotes
### Proposal: Add Game of Thrones Quotes Block

**Description:**
I would like to propose adding a "Game of Thrones" quotes block to the project. This block could showcase iconic quotes from the series, enhancing the content and engaging fans of the show.
The show's memorable quotes encapsulate its themes of power, betrayal, and survival 

**Implementation Suggestions:**
- Create a new component for displaying the quotes.

### PR DESCRIPTION
**Pull request by HritvikBhatia**
Proposal: #76 

Changes Made
Added a JSON file containing 100 quotes from Game of Thrones.
Created a display card section to render the quotes on the page.

Screenshot 2024-10-10 083312

closes #76

### CODE REVIEW

**Review by akanshSirohi — CHANGES_REQUESTED** [review]
@HritvikBhatia Please resolve conflicts in your PR

**Comment by HritvikBhatia** [comment]
@akanshSirohi done sir

**Review by akanshSirohi — CHANGES_REQUESTED** [review]
@HritvikBhatia Please add your card to readme.md too.

**Comment by HritvikBhatia** [comment]
@akanshSirohi done sir 😸😺

File: README.md
Comment (akanshSirohi) [inline README.md:1]: @HritvikBhatia  Check the line 67, you have added some extra text ":wq" looks like you were trying to use vim. I have fixed it but please check your PR from next time properly.

**Review by akanshSirohi — APPROVED** [review]

**Pull request merged**